pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -203,9 +212,18 @@ public static void main(String[] args) throws Exception {
         if (arguments.subscriptionType != SubscriptionType.Exclusive &&
                 arguments.subscriptions != null &&
                 arguments.subscriptions.size() != arguments.numConsumers) {
-            System.out.println("The size of subscriptions list should be equal to --num-consumers when subscriptionType isn't Exclusive");
-            jc.usage();
-            System.exit(-1);
+            // keep compatibility with the previous version
+            if (arguments.subscriptions.size() == 1) {
+                List<String> defaultSubscriptions = Lists.newArrayList();
+                for (int i = 0; i < arguments.numSubscriptions; i++) {
+                    defaultSubscriptions.add(String.format("%s-%d", arguments.subscriberName, i));
```

Comment:
Same as the above comment, only apply the numSubscriptions when > 1;

Reply:
Same as above since the default initial value is `sub` which is same as `arguments.subscriberName`.